### PR TITLE
Persist wireguard peers

### PR DESCRIPTION
**Pull request by neacsu**

### CODE REVIEW

File: common/gateway-storage/src/models.rs [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+                }
+            }),
+            tx_bytes: value.tx_bytes as i64,
+            rx_bytes: value.rx_bytes as i64,
+            persistent_keepalive_interval: value.persistent_keepalive_interval.map(|v| v as i64),
+            allowed_ips: bincode::Options::serialize(
+                bincode::DefaultOptions::new(),
+                &value.allowed_ips,
+            )
+            .unwrap(),
```
Comment (octol) [inline common/gateway-storage/src/models.rs]: Are we really really sure this is safe? I will sleep better at night if the unwrap is removed ;)

File: nym-node/src/config/template.rs [hunk 2]
```diff
@@ -221,6 +221,10 @@ upstream_exit_policy_url = '{{ exit_gateway.upstream_exit_policy_url }}'
 
 [exit_gateway.storage_paths]
 
+# Path to sqlite database containing all persistent data: messages for offline clients,
```
Comment (octol) [inline nym-node/src/config/template.rs:224]: Is this description correct? I thought we removed messages for offline clients?

**Review by octol — APPROVED** [review]

File: nym-node/src/config/template.rs (hunk 2 above)
Comment (neacsu) [reply nym-node/src/config/template.rs:224]: I'm not sure tbh. If you know offline clients are removed, I can modify the description too, I just modified the wireguard part

**Comment by jstuczyn** [comment]
I see you've done some restructuring inside the gateway. would you mind waiting with merging this PR until I could get ecash in? I really don't want to be dealing with more rebasing on my end anymore : (

**Comment by neacsu** [comment]
> I see you've done some restructuring inside the gateway. would you mind waiting with merging this PR until I could get ecash in? I really don't want to be dealing with more rebasing on my end anymore : (

No worries, let me know when you get it merged, it's been a while since I've done a rebasing :D

**Pull request merged**